src/settings.c

```diff
@@ -36,6 +36,9 @@ pwquality_default_settings(void)
         pwq->dict_check = PWQ_DEFAULT_DICT_CHECK;
         pwq->user_check = PWQ_DEFAULT_USER_CHECK;
         pwq->enforcing = PWQ_DEFAULT_ENFORCING;
+	pwq->retry_times = PWQ_DEFAULT_RETRY_TIMES;
+	pwq->enforce_for_root = PWQ_DEFAULT_ENFORCE_ROOT;
+	pwq->local_users_only = PWQ_DEFAULT_LOCAL_USERS;
```

Comment:
Please align the indentation here.

Reply:
I guess I missed a few as I thought I had made sure the indentation was right. Oh well. Fixed.